grommet/grommet
# add back in scroll for select

#### What does this PR do?
This PR adds a  `useEffect` that looks for the position of the  `activeIndex` to bring the behavior of scroll for keyboard
#### Where should the reviewer start?
selectContainer.js
#### What testing has been done on this PR?
keyboard on storybook 

any of longer options  in `select` for example many options use down arrow and the scroll will appear when you  get to the bottom of the container
#### How should this be manually tested?
**many options story** use down arrow and the scroll will appear when you  get to the bottom of the container
#### Any background context you want to provide?
This code was previously here in `select` in  V2.12.0  however this was taken out because of the new behavior that  was added into `infiniteScroll` at the time this took care of the keyboard scroll. However when `infiniteScroll` behavior changed to `setState` for  `showProp` this no longer updated the `show` after the first render. 

https://github.com/grommet/grommet/blob/master/src/js/components/InfiniteScroll/InfiniteScroll.js#L156


#### What are the relevant issues?
closes #5231 
#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
no
#### Should this PR be mentioned in the release notes?
yes
#### Is this change backwards compatible or is it a breaking change?
backwards compatible
